$NetBSD$

WARN: PLIST:105: Man pages should be installed into man/, not share/man/.

--- apps/CMakeLists.txt.orig	2013-09-03 04:17:12.000000000 +0900
+++ apps/CMakeLists.txt	2013-09-04 23:17:24.000000000 +0900
@@ -121,5 +121,5 @@ if(NOT APPLE AND NOT WIN32)
 add_definitions("-DCONFDIR=\"${CMAKE_INSTALL_PREFIX}/share/TrustedQSL/\"")
 install(TARGETS tqsl DESTINATION bin)
 install(DIRECTORY help DESTINATION share/TrustedQSL)
-install(FILES tqsl.5 DESTINATION share/man/man5)
+install(FILES tqsl.5 DESTINATION man/man5)
 endif()
